src/index.ts

```diff
@@ -3,5 +3,5 @@ import {extract} from './text';
 
 module.exports = {
```

Comment:
Since this is typescript file, let's export these functions in a typescript way `export ...`